internal/server/grpc.go

```diff
@@ -62,6 +63,11 @@ func newGrpcServer(opts *options) (*grpcServer, error) {
 		)
 	}
 
+	if opts.TelemetryEnabled {
```

Comment:
Might add a comment here that this is the only place we wire telemetry into our grpc server.